### ISSUE TITLE
Missing publisher in JIS

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-jis#15]
I need to know from the publisher field whether JIS references are published by JIS, for proper rendering in JIS https://github.com/metanorma/mn-samples-jis/issues/26 . But at least JIS C 0450 does not have a publisher:

````
        <contributor>
          <role type="author"/>
          <organization>
            <name language="ja" script="Jpan">一般財団法人　日本規格協会</name>
          </organization>
        </contributor>
````

Could you please add a publisher matching the author consistently? 

I'd also like an English name for JIS as well as a Japanese one, but I gather that name/variant is giving you difficulty...

**Comment by opoudjis** [comment]
The publisher is "JSA":

* Japanese Standards Association (English)
* 日本規格協会 (Japanese)

**Comment by ronaldtse** [comment]
The publisher is "JSA":

* Japanese Standards Association (English)
* 日本規格協会 (Japanese)

**Comment by andrew2net** [comment]
> I'd also like an English name for JIS as well as a Japanese one, but I gather that name/variant is giving you difficulty...

@opoudjis `organization/nae` one or more cardinality, so we don't have to use variants here

**Comment by andrew2net** [comment]
Fixed in v1.19.1
